LCM/dsc/engine/ConfigurationManager/LocalConfigurationManager.c

```diff
@@ -2444,7 +2530,22 @@ void Invoke_SendConfiguration(
     {      
         args->force = MI_TRUE;
     }    
-    Thread_CreateDetached(Invoke_SendConfigurationApply_Internal, NULL, args); 
+
+    {
+	LCMTaskNode * newNode = (LCMTaskNode*)DSC_malloc(sizeof(LCMTaskNode), NitsHere());
```

Comment:
It could be, but keeping in style with the rest of this file, there's lots of duplicate code.